src/modules/intelligence/application/use_cases/translate_article.py

```
@@ -0,0 +1,245 @@
+            analysis_id=analysis_id,
+            language=target_language,
+            summary=translated.summary,
+            pain_points=translated.pain_points,
+            insights=translated.insights,
+            innovations=translated.innovations,
+        )
+
+        try:
+            self._translation_repository.save(translation)
+            logger.info("translation_saved", analysis_id=str(analysis_id), language=target_language)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)